amplipi/streams.py

```diff
@@ -1331,7 +1341,9 @@ def _activate(self, vsrc: int):
           # NOTE: port 9000 is assumed
           server.replace('localhost', socket.gethostname())
         lms_args += ['-s', server]
-      # TODO: allow port to be specified with server (embedding it in the server URL does not work)
+
+      meta_args = ['python3', 'streams/lms_metadata.py', '--name', self.name, '--server', self.server]
```

Comment:
it looks like the thing mypy is complaining about is the `Optional` bit; i think it may be because self.name and self.server are not guaranteed to be set here, which could indeed trigger a bug